### PR TITLE
1.24.1 release notes

### PR DESCRIPTION
**Pull request by dhawton**
## Description

1.24.1 release notes, will cherrypick to master

## Reviewers

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Review by bleggett — APPROVED** [review]
looks correct to me.

File: content/en/news/releases/1.24.x/announcing-1.24.1/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  AppArmor profiles which block certain privileged pod capabilities. Previously, AppArmor
+  (when enabled) was bypassed for the `istio-cni-node` `DaemonSet` since privileged was set to true
+  in the `SecurityContext`. This change ensures that the AppArmor profile is set to unconfined
+  for the `istio-cni-node` `DaemonSet`.
+
+- **Added** `dnsPolicy` of `ClusterFirstWithHostNet` to `istio-cni` when it runs with `hostNetwork=true` (i.e. ambient mode).
+
+- **Fixed** an issue where `istioctl install` was not working as expected on Windows.
+
+- **Fixed** an issue where merging `Duration` with an `Envoyfilter` can lead to all listeners associated attributes unexpectedly became modified
```
Comment (mikemorris) [inline content/en/news/releases/1.24.x/announcing-1.24.1/index.md:26]:
```suggestion
- **Fixed** an issue where merging `Duration` with an `EnvoyFilter` can lead to all listeners associated attributes being unexpectedly modified
```

**Review by mikemorris — APPROVED** [review]
Minor phrasing nit, aside from that release notes LGTM.

File: content/en/news/releases/1.24.x/announcing-1.24.1/index.md (hunk 1 above)
Comment (mikemorris) [reply content/en/news/releases/1.24.x/announcing-1.24.1/index.md:26]: I'm also not quite clear on whether `listeners` is intended to be possessive, plural or both - not a huge concern but someone more familiar with this change may want to re-read the note for accuracy.

**Review by craigbox — APPROVED** [review]

**Review by kfaseela — APPROVED** [review]

**Pull request merged**